charts/rollout-app/templates/prometheusrules.yaml

```diff
@@ -163,59 +163,6 @@ spec:
   - name: {{ .Release.Namespace }}.{{ .Release.Name }}.{{ .Chart.Name }}.DeploymentRules
     rules:
 
-    {{- with .Values.prometheusRules.DeploymentGenerationMismatch }}
```

Comment:
This alarm never made sense here. These are `Rollout` objects, the `kube_deployment...` metrics simply do not exist for them!